package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "jshint": "^2.9.6",
```

Comment:
@Musacoli , could there be any specific reason why you added `jshint`??

Reply:
None shall have it removed now